samples/bluetooth/hci_rpmsg/nrf5340_cpunet_iso_broadcast.conf

```diff
@@ -22,22 +22,27 @@ CONFIG_BT_OBSERVER=n
 CONFIG_BT_CENTRAL=n
 CONFIG_BT_EXT_ADV=y
 CONFIG_BT_PER_ADV=y
-CONFIG_BT_ISO_BROADCASTER=y
 # CONFIG_BT_PER_ADV_SYNC=y
+CONFIG_BT_ISO_BROADCASTER=y
 # CONFIG_BT_ISO_SYNC_RECEIVER=y
+CONFIG_BT_ISO_MAX_CHAN=2
 
 # ISO Broadcast Controller
-CONFIG_BT_CTLR_ADV_EXT=y
 CONFIG_BT_CTLR_ADV_PERIODIC=y
 CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=191
 CONFIG_BT_CTLR_ADV_ISO=y
 CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX=251
-CONFIG_BT_CTLR_ISO_TX_BUFFERS=3
+CONFIG_BT_CTLR_ISO_TX_BUFFERS=16
```

Comment:
This is a significant increase. Since we are just doubling the number of channels, shouldn't this only be doubled as well?

Reply:
These numbers where required on the nrf5340 audio dk gateway where the audio is sourced over USB, and many samples arrive before ISO interval.